docs/portal/registry-users/updating-users.md

```diff
@@ -0,0 +1,16 @@
+# Updating Users
+
+Updating a user allows admin portal users to change a user's email address, description, access to projects, and access type. To make sure the appropriate user is updated, a user's information can be displayed before the user is updated.
+**Note:** Non-admin portal users are not permitted to upgrade users.
+1. In the left pane, click **Image Registry > Registry Users**.
+2. Perform one of the following tasks:<ul><li>To update a user from the Registry Users page, click the user's **Actions** button and select **Update**. </ul></li> <ul><li>To view information about a user before updating the, click the username to display the Edit page. To update the user, click the **Update** button at the top right.</ul></li>
+      ![null](</docs/resources/images/registry/update-user.png>)
+3. Change the settings as necessary.
+   
+| **Field**              | **Description**                                                            |
+| -----------------------|----------------------------------------------------------------------------| 
+| Username               | Read-only field that shows the name of this user.                          |
+| Email                  | Email address of the registry user.                                        |
+| Description            | Optional description about the registry user (up to 20 characters)         |
+| Access to Projects     | Check the projects that will be accessible to the registry user and uncheck the ones that will not be accessible. For accessible projects, select whether the user can view the project, but not edit it (**readOnly**) or view and edit the project (**readWrite**).|
```

Comment:
Again, readOnly means a registry user is allowed to pull images from project registry, while readWrite means both pulling and pushing images are allowed.

Reply:
Done.